### PR TITLE
Sunset the plugin

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Foreman OpenNebula Plugin
 
+**This plugin has been discontinued. The latest release only helps with its removal. If you're interested in taking over the maintanance, let us know.**
+
+In order to remove the plugin from you Foreman installation, following steps need to be taken
```
Comment (mmoll) [inline README.md]:
```suggestion
In order to remove the plugin from your Foreman installation, the following steps need to be taken:
```

**Review by mmoll — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
 # Foreman OpenNebula Plugin
 
+**This plugin has been discontinued. The latest release only helps with its removal. If you're interested in taking over the maintanance, let us know.**
```
Comment (ekohl) [inline README.md]: Might be good to post to discourse and link it here so users know where to start.

**Comment by ares** [comment]
announced and incorporated both comments

**Comment by mmoll** [comment]
merged, díky @ares!
@ekohl could you archive the repo?

**Pull request merged**